stonesoup/simulator/tests/test_platform.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+
+import numpy as np
+
+from ...types.state import State
+
+from ..platform import PlatformDetectionSimulator
+from ..simple import SingleTargetGroundTruthSimulator
+
+
```

Comment:
Suggest convenience method with similar functionality to your platform class.

```suggestion
def build_platform(sensors, x_velocity):

    state = State(state_vector=StateVector([[0], [x_velocity], [0], [0]]),
                  timestamp=datetime.datetime(2020, 4, 1))
    model_1d = ConstantVelocity(0.0)  # zero noise so pure movement
    trans_model = CombinedLinearGaussianTransitionModel([model_1d] * 2)
    mounting_offsets = np.zeros((len(sensors), 2))
    mounting_mappings = np.array([[0, 2]])
    platform = SensorPlatform(state=state, sensors=sensors, transition_model=trans_model,
                              mounting_offsets=mounting_offsets,
                              mounting_mappings=mounting_mappings)
    return platform

```

Then replace `platform_class` with `build_platform` throughout the below.